fe/fe-core/src/main/java/org/apache/doris/catalog/BuiltinScalarFunctions.java

```diff
@@ -616,6 +622,12 @@ public class BuiltinScalarFunctions implements FunctionHelper {
             scalar(StPolygonfromtext.class, "st_polygonfromtext"),
             scalar(StX.class, "st_x"),
             scalar(StY.class, "st_y"),
+            scalar(StGeometryFromWKB.class, "st_geometryfromwkb"),
```

Comment:
please add function in lexicographical order

Reply:
done